src/core/lang.c

```diff
@@ -295,6 +311,10 @@ const uint8_t *lang_get_string(int group, int index)
     if (group == 96 && !index) {
         return translation_for(TR_BUILDING_SMALL_TEMPLE_VENUS_NAME);
     }
+    //TODO: Move to proper group and translation
+    if (group == 97 && !index) {
+        return (uint8_t*) "Go to the site";
```

Comment:
Hardcoded temporarily

Reply:
@miiila FYI, this was indeed incorrect.

You added this line here, which made the "market" building information window display the title "Go to the site".

Instead of short-circuiting `lang_get_string` when using `lang_text_draw` you should have used `text_draw(translation_for(TR_GO_TO_THE_SITE`.